packages/components/table/src/table-footer/index.ts

```diff
@@ -117,19 +124,8 @@ export default defineComponent({
                     key: cellIndex,
                     colspan: column.colSpan,
                     rowspan: column.rowSpan,
-                    class: [
-                      ...this.getRowClasses(column, cellIndex),
-                      'el-table__cell',
-                      hasGutter && cellIndex === columns.length - 1
-                        ? 'last'
-                        : '',
-                    ],
-                    style: this.getRowStyles(
-                      column,
-                      cellIndex,
-                      hasGutter,
-                      gutterWidth
-                    ),
+                    class: getCellClasses(columns, cellIndex, hasGutter),
+                    style: getCellStyles(column, cellIndex, hasGutter),
```

Comment:
**Ideas:** I just came up with an idea that `getCellClasses` and `getCellStyles` are computed every single time, and the value might not be the same, not familiar with how `Vue` compares them(maybe we should dig it in), but we can cache this value

We can do this later.